content/kbld/docs/latest/config.md

```diff
@@ -158,6 +160,26 @@ kubectl create secret docker-registry buildkit --docker-server=https://index.doc
 
 See project site for details: [buildkit-cli-for-kubectl](https://github.com/vmware-tanzu/buildkit-cli-for-kubectl).
 
+### ko
+
+Available as of v0.28.0+
+
+```yaml
+---
+apiVersion: kbld.k14s.io/v1alpha1
+kind: Config
+sources:
+  - image: image1
```

Comment:
nit: unindent by 2 spaces. we typically do not indent array items in docs/examples.